Update Agent Pools scope to Workspaces (#870)
hashicorp/terraform-provider-tfe: website/docs/d/agent_pool.html.markdown

In addition to all arguments above, the following attributes are exported:

* `id` - The agent pool ID.

> * `organization_scoped` - Whether or not the agent pool is scoped to all workspaces in the organization.

Reviewer: ```suggestion
* `organization_scoped` - Whether or not the agent pool can be used by all workspaces in the organization.
```

> * `allowed_workspace_ids` - IDs of the workspaces that are allowed to use the agent pool.

Reviewer: ```suggestion
* `allowed_workspace_ids` - If `organization_scoped` is set to `false`, the IDs of the workspaces that are allowed to use the agent pool.
```
